rosidl_typesupport_introspection_cpp/include/rosidl_typesupport_introspection_cpp/MessageIntrospection.h

```diff
@@ -6,7 +6,7 @@
 
 #include <rosidl_generator_c/message_type_support.h>
 
-#include "impl/visibility_control.h"
+#include "visibility_control.h"
```

Comment:
Should `visibility_control.h` be moved on level up out of the impl folder and be included with its namespace? That would also make the exported include directory `include/${PROJECT_NAME}/impl` obsolete.